client/js/components/make-sticky.js

```diff
@@ -1,25 +1,43 @@
 import {nodeList, featureTest} from '../util';
-import debounce from 'lodash.debounce';
-
-const makeSticky = (els) => {
-  if (featureTest('position', 'sticky')) {
-    const elements = nodeList(els);
-    const applyPositioning = () => {
-      if (document.readyState === 'complete') {
-        elements.forEach((e) => {
-          if (e.offsetHeight > window.innerHeight) {
-            e.classList.remove('sticky-applied');
-          } else {
-            e.classList.add('sticky-applied');
-          }
-        });
-      }
-    };
-
-    document.addEventListener('readystatechange', applyPositioning);
-    window.addEventListener('resize', debounce(applyPositioning, 500));
-    window.addEventListener('orientationchange', applyPositioning);
-  }
+import dropRepeats from 'xstream/extra/dropRepeats';
+import {windowOrientationChange$, windowResize$} from '../utils/dom-events';
+import sampleCombine from 'xstream/extra/sampleCombine';
+
+const initialPxFromTop = 15; // TODO: remove magic
+
+const makeSticky = (els, store$) => {
+  if (!featureTest('position', 'sticky')) return;
+
+  const elements = nodeList(els);
+
+  const applyPositioning = (stickyNavHeight) => {
+    elements.forEach((el) => {
```

Comment:
I believe it was there to ensure that the to-be sticky image's heights were calculated once they'd loaded.